megamek/src/megamek/common/Dropship.java

```diff
@@ -1869,4 +1869,13 @@ public void newRound(int roundNumber) {
     public void postProcessFacingChange() {
         mpUsed += 2;
     }
+
+    /**
+     * Depsite being able to hover, aerodyne dropships are
+     * explicitely forbidden from vertical landings.
+     */
+    @Override
+    public boolean canLandVertically() {
```

Comment:
This isn't quite correct: per TW page 88, aerodyne dropships and small may land vertically in vacuum.